src/cmd/manage_nsfs.js

```diff
@@ -103,9 +103,7 @@ async function main(argv = minimist(process.argv.slice(2))) {
                 ...(ManageCLIError.FS_ERRORS_TO_MANAGE[err.code] ||
                 ManageCLIError.RPC_ERROR_TO_MANAGE[err.rpc_code] ||
                 ManageCLIError.InternalError), cause: err });
-        process.stdout.write(manage_err.to_string() + '\n', () => {
-            process.exit(1);
-        });
+        write_stdout_response(manage_err);
```

Comment:
I think some of the events have been recorded two times with this. So, we can just create an event for `InternalError` only. wdyt?